macho.c

```diff
@@ -0,0 +1,1357 @@
+/* macho.c -- Get debug data from an Mach-O file for backtraces.
+   Copyright (C) 2012-2016 Free Software Foundation, Inc.
+   Written by Ian Lance Taylor, Google.
+   Copyright (C) 2017 John Colanduoni.
```

Comment:
I can't accept this change with an additional copyright statement like this.  You can add your name, as in pecoff.c (and you don't need to put my name here), but I can only accept changes that are only copyright Free Software Foundation.